crates/wasi-http/src/lib.rs

```diff
@@ -95,29 +95,29 @@ impl TryFrom<http::Request<Bytes>> for Request {
     }
 }
 
-pub struct Method<'a>(http_types::MethodParam<'a>);
+pub struct Method(http_types::Method);
```

Comment:
Do we need to make a new type that Derefs to Method? Instead, we could just `impl TryFrom<http::Method> for http_types::Method`, and not create a new type at all.

Reply:
i am assuming that the preview2 interfaces will be imported from the crate `wasi`